docs/source/user/aerodyn-olaf/RunningOLAF.rst

```diff
@@ -17,80 +16,249 @@ available in the :ref:`installation` documentation.
+The approximate time needed for the wake to reach a desired downstream distance :math:`d_\text{target}` is therefore:
+
+
+.. math:: 
+   
+    T_\text{target} = \frac{d_\text{target}}{U_0(1-k_a a)}
+
+This time corresponds to a number of time steps (i.e. the total number of wake panels) equal to:
+
+.. math:: :label: ntargetDist
+
+   
+    n_\text{target,distance} 
+       =  \frac{T_\text{target}}{\Delta t_\text{FVW}} 
+       =  \frac{d_\text{target}}{U_0(1-k_a a) \Delta t_\text{FVW}} 
+       \approx \frac{d_\text{target}}{0.5 U_0 \Delta t_\text{FVW}} 
+       \approx \frac{12 D}{U_0 \Delta t_\text{FVW}} 
+       \qquad \text{(integer)}
+
+where the first approximation uses :math:`k_a a\approx 0.5` and the second approximation assumes a target distance of 6D.
+It is also possible to define the number of near wake panels based on a total number of revolution, :math:`n_\text{rot}`, leading to:
+
+.. math:: :label: ntargetRot
+   
+    n_\text{target, rotations} 
+       = \frac{n_\text{rot} T_\text{rot}}{dt_\text{FVW}}
+       = \frac{n_\text{rot} 2 \pi }{\Omega dt_\text{FVW}}
+       = \frac{n_\text{rot} 60 }{\Omega_\text{RPM} dt_\text{FVW}}
+       \qquad \text{(integer)}
+
+
+The wake of OLAF consists of two regions defined as "near wake" (NW) and "far wake" (FW), where the far consists of rolled-up tip and root vortices. 
+The far wake has reduced accuracy, therefore  velocity profiles crossing the far wake will likely be inaccurate. The role of the far wake is to reduce the computational time. For increases accuracy, it is recommended to use a longer near wake and a shorter far wake. The far-wake may be removed altogether.
+The far wake is further decomposed into two regions, a region where the vortex filaments are free to convect, and another one where the filaments convect with a fixed, averaged induced velocity. The advantage of having this "fixed" far wake region, is that it mitigates the impact of wake truncation  which is an erroneous boundary condition (vortex lines cannot end in the fluid). If the wake is truncated while still being "free", then the vorticity will rollup in this region. It is therefore useful for have this fixed far-wake region, the free far-wake region is not as important (in the future, OLAF will introduced a fixed near-wake region, so that the far-wake can be omitted altogether).
```

Comment:
Change "for have this" to "to have a" and change "region, the" to "region. The".

Reword the parenthetical comment to be something like, "(in the future, a fixed near-wake region could be introduced in OLAF so that the far wake can be omitted altogether)".